.claude/skills/bug-dump-ingest/SKILL.md

```
@@ -0,0 +1,542 @@
+---
+name: bug-dump-ingest
+description: 'Syncs the #bug-dump Slack channel into Linear as the system of record. Primary mode: bulk sync — every ingestable message becomes a Linear issue with labels (area, env, severity, reporter) in Triage state, marked in Slack via a :white_check_mark: reaction on the parent + a thread reply carrying the Linear URL. Respects the team emoji scheme (:white_check_mark: ticket created, :pr-open: PR open, :question: needs context, :repeat: duplicate). Secondary mode: delegate to red-green-fix for explicitly-approved fixes. Use when asked to sync #bug-dump to Linear, triage slack bugs, run a bug-dump sweep, or ingest bug reports. Triggers on: bug-dump, sync bug-dump, ingest bugs, triage slack bugs, bug sweep.'
+---
+
+# Bug Dump Ingest
+
+**Primary job: sync `#bug-dump` (Slack: `C0A4XMHANP3`) into Linear as the source of truth.** Linear is where status, labels, and follow-up triage happen — this skill just gets every bug into Linear with enough context that a downstream agent or human can work from Linear alone, not from Slack.
+
+Fix workflows (red-green tests, PRs) are a **secondary, opt-in pass** triggered per-candidate with `F`. The default flow files tickets and stops.
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)